src/plugins/dataflow/components/dataflow-program.tsx

```diff
@@ -801,7 +802,8 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
     const { deviceFamily } = this.stores.serialDevice;
 
     if (deviceFamily === "arduino" && isNumberOutput){
-      this.stores.serialDevice.writeToOutForBBGripper(n.data.nodeValue as number);
+      const liveOutputType = n.controls.get("hubSelect")?.getData("liveOutputType") as string;
```

Comment:
Thanks - you are right, this should not be allowed to call `writeToOutForBBGripper` without the proper params.  It's interesting that this points to the same issue as above - writeToOutForBBGripper is looking for one of those output type `name` values.  Anyway, since `dataflow-program` already does a bunch of checking for these names, I added a check to be sure we have one of them in hand before calling the serial function.